drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -880,6 +885,16 @@ local function wind_support_handler(driver, device, ib, response)
       table.insert(supported_wind_modes, wind_mode.NAME)
     end
   end
+  -- this check re-profiles devices with a feature map indicating they support WindMode,
+  -- but with a WindSupport value of 0 in their device specifications.
+  if #supported_wind_modes < 2 then
+    local fixed_profile_name = string.gsub(device:get_field(PROFILE_NAME), "-wind", "")
+    if fixed_profile_name == "fan" then
+      fixed_profile_name = "fan-generic"
```

Comment:
I don't see this `fan-generic` profile on main or the matter-v1.3-drivers branch, is that something that is needed here?

Reply:
Yes sorry, I preemptively changed it since this will go out after the changes in [this PR](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1578), which is going to deprecate the fan profile, since our current fan profile is actually something more akin to fan-wind.